itests/start-chronos.sh

```diff
@@ -1,4 +1,4 @@
 sudo /etc/init.d/zookeeper start
 sleep 5
 sudo /etc/init.d/chronos start
-sleep 5
+sleep infinity
```

Comment:
This can't sleep forever, it needs to return right away to allow the next step (tox) to go